src/main/kotlin/no/nav/familie/ef/sak/service/DbMigreringService.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.familie.ef.sak.service
+
+import no.nav.familie.ef.sak.repository.TilkjentYtelseRepository
+import org.slf4j.LoggerFactory
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.util.*
+
+@Service
+class DbMigreringService(private val dbMigreringRepository: TilkjentYtelseRepository) {
+
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+    @Scheduled(initialDelay = 120000, fixedDelay = ÅR)
+    @Transactional
+    fun dbMigrering() {
+        val andeler = dbMigreringRepository.findAll()
```

Comment:
Skal denne slettes etter å ha gått ut i prod? Kjøres vel på hver "oppstart" nå?

Reply:
Yep, gjøres i del 2 https://github.com/navikt/familie-ef-sak/pull/371

Reply:
Denne blir vel kjørt 2 ganger, en gang per pod, men er vel egentlige greit.

Reply:
Det går fint at den kjøres to ganger, var mer at vi ikke trenger den "til evig tid" :)